.github/workflows/tests.yml

```diff
@@ -64,6 +64,12 @@ jobs:
       - name: "Build libfake"
         run: ./bin/inv_wrapper.sh libfake
         working-directory: ${{ github.workspace }}/clients/cpp
+      - name: "Update ImageMagick submodule"
+        run: git submodule update --init -f third-party/ImageMagick
+        working-directory: ${{ github.workspace }}/clients/cpp
```

Comment:
Why do we need to update the ImageMagick submodule here? We're checking out the code with `submodules: true` which should do this implicitly.